addons/survey/static/src/views/fields/boolean_toggle/survey_boolean_toggle_field.js

```diff
@@ -0,0 +1,14 @@
+/** @odoo-module **/
+import { registry } from "@web/core/registry"
+import { BooleanToggleField } from '@web/views/fields/boolean_toggle/boolean_toggle_field';
+
+export class SurveyBooleanToggleField extends BooleanToggleField {
```

Comment:
Maybe something specifying that we don't save could help here as well.
```suggestion
export class SurveyBooleanToggleFieldNoSaving extends BooleanToggleField {
```

Reply:
Hey, @flch-odoo 
Done with all suggested changes!! Can you please check it out? 
Thanks:)